client/asset/dcr/rpcwallet.go

```diff
@@ -72,6 +87,10 @@ type rpcWallet struct {
 	// or a stub for testing.
 	rpcClient rpcClient
 
+	// Calling w.rpcClient.GetTickets too close together will return no
+	// tickets for the second call. A ticket cache alleviates this.
```

Comment:
It's a dcrwallet bug?

Reply:
Unsure...

Reply:
I think I was wrong about this. Removing for now.